src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -843,10 +843,6 @@ FixedwingPositionControl::control_auto(const hrt_abstime &now, const Vector2d &c
 	_att_sp.pitch_reset_integral = false;
 	_att_sp.yaw_reset_integral = false;
 
-	if (pos_sp_curr.valid && pos_sp_curr.type == position_setpoint_s::SETPOINT_TYPE_LOITER) {
-		publishOrbitStatus(pos_sp_curr);
-	}
-
 	position_setpoint_s current_sp = pos_sp_curr;
```

Comment:
@Jaeyoung-Lim are you sure? it looks like it is a full position setpoint type here? or do I misunderstand something? (sorry for not linking the line number before)

Reply:
@tstastny Sorry, you are right! Fixed in last commit